### PR TITLE
WIP: Bug 535820 - JDK 11 CORBA Support

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
PR 156 replacement

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=535820

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=535820

**Comment by Tomas-Kraus** [comment]
It's buildable and tests are passing. We need CQs for all new dependxencies.

**Comment by Tomas-Kraus** [comment]
I need to update org.glassfish.pfl.pfl* too because I made some nasty change in one of the manifests to get rid of sun.misc issue with P2 repository. Now it's solved so I can use original packages.

File: buildsystem/compdeps/org.eclipse.persistence.compdeps/feature.xml [hunk 1]
```diff
@@ -45,6 +45,111 @@
...
+    <plugin
+            id="org.glassfish.corba.exception-annotation-processor"
+            download-size="0"
+            install-size="0"
+            version="@GF_CORBA_EAP_VERSION@"
+            unpack="false"/>
+
+    <plugin
+            id="org.osgi.core"
+            download-size="0"
```
Comment (lukasj) [inline buildsystem/compdeps/org.eclipse.persistence.compdeps/feature.xml]: Why import plugin (as in features/ope.jpa/feature.xml cannot be used here instead?

File: antbuild.properties [hunk 2]
```diff
@@ -72,6 +72,7 @@ eclipselink.antlr=${eclipselink.plugins}/${antlr}
 eclipselink.mvn.parent=${eclipselink.build.base}/org.eclipse.persistence.parent
 eclipselink.compdeps=${eclipselink.build.base}/compdeps
 eclipselink.oraclelibs=${eclipselink.build.base}/oraclelibs
+eclipselink.sun-misc=${eclipselink.compdeps}/sun.misc
```
Comment (lukasj) [inline antbuild.properties]: If I start java with "-Dorg.osgi.framework.system.packages.extra=sun.misc" (osgi 4.3+) then I don't need this bundle. What about just documenting this?

**Comment by Tomas-Kraus** [comment]
Thanks for the feedback, I'll do it.

**Comment by lukasj** [comment]
to check maven artifacts, you can do `ant -f antbuild.xml build-distribution` followed by sth like `ant -f uploadToNexus.xml -DsnapshotId=local -DsnapshotURL=file:///some/path/m2-repo -Drelease.version=3.0.0 -Dbuild.type=SNAPSHOT -Dgit.hash=local -Dmavenant.dir=/folder/with/maven-ant-tasks-2.1.3.jar -Dversion.string=3.0.0.qualifier

**Comment by Tomas-Kraus** [comment]
CQs:
- Glassfish CORBA: http://dev.eclipse.org/ipzilla/show_bug.cgi?id=17026
- Glassfish PFL: http://dev.eclipse.org/ipzilla/show_bug.cgi?id=17027
- org.osgi.core: http://dev.eclipse.org/ipzilla/show_bug.cgi?id=17028
- GMBAL: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17036
- management-api: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=17037

File: antbuild.properties (hunk 2 above)
Comment (Tomas-Kraus) [reply antbuild.properties]: I already made those changes. We may keep them in some branch and point documentation on it.

**Comment by lukasj** [comment]
Compdeps part is in pr #272, the PR for the rest will be submitted once that is integrated

**Pull request closed without merging**